pkg/k8s/watchers/cilium_endpoint.go

```diff
@@ -38,7 +38,7 @@ type k8sCiliumEndpointsWatcherParams struct {
 	K8sAPIGroups      *k8sSynced.APIGroups
 
 	EndpointManager endpointmanager.EndpointManager
-	PolicyUpdater   *policy.Updater
+	PolicyUpdater   policyCell.PolicyUpdater
```

Comment:
From a code organization point of view - it strikes me as strange that we have a package specifically for cell components and we also have some policy interface types in there.

Reply:
But I don't know the context here, so please disregard if there's a good reason for this.

Reply:
Was thinking about this as well. Typically we may have a `types` sub-package, haven't seen too many `cell` sub-packages, but maybe they serve similar function: to avoid importing the "implementation" with all its dependencies? If so, maybe collapse them into a single `types` package?